Remotion/Web/Development.WebTesting.TestSite.Shared/GenericTest.aspx.cs

```diff
@@ -18,19 +18,27 @@
 using System.Collections.Generic;
 using System.Web.UI;
 using System.Web.UI.WebControls;
+using Remotion.ServiceLocation;
 using Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure;
-using Remotion.Web.Development.WebTesting.TestSite.GenericPages;
 using Remotion.Web.Development.WebTesting.TestSite.Infrastructure;
+using Remotion.Web.Development.WebTesting.TestSite.Shared.GenericPages;
 using Remotion.Web.UI.Controls;
+using GenericTestPageParameter = Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure.GenericTestPageParameter;
 
-namespace Remotion.Web.Development.WebTesting.TestSite
+namespace Remotion.Web.Development.WebTesting.TestSite.Shared
 {
   public partial class GenericTest : GenericTestPageBase<GenericTestOptions>
   {
     protected override Dictionary<string, GenericTestPageParameter> Parameters { get; } = new Dictionary<string, GenericTestPageParameter>();
 
+    public string FrameTestFrameUrl { get; }
+
     public GenericTest ()
     {
+      FrameTestFrameUrl = SafeServiceLocator.Current.GetInstance<IResourceUrlFactory>()
+          .CreateResourceUrl(typeof(GenericTestFunction), TestResourceType.Root, "FrameTestFrame.aspx")
+          .GetUrl();
+
```

Comment:
Given the way ASP.net works, it would be cleaner to make this lazy (and skip caching) and just have the property return the URL. Or we get a helper function that takes the file and trasforms it.
```
public static class Resolve
{
 string Root(string file);
 string Image(string file);
 string Html(string file);
}
```
(just an idea for a simple syntax)